packages/ISO8601/ISO8601.0.2.5/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "1.2"
+
+available: [ ocaml-version >= "3.12.1" ]
+
+version: "0.2.5"
+
+name: "ISO8601"
+
+authors: [ "Julien Sagot" ]
+
+maintainer: "ju.sagot@gmail.com"
+
+homepage: "http://github.com/sagotch/ISO8601.ml/"
+
+dev-repo: "https://github.com/sagotch/ISO8601.ml.git"
+
+doc: "http://sagotch.github.io/ISO8601.ml/"
+
+bug-reports: "https://github.com/sagotch/ISO8601.ml/issues"
+
+build-doc: [ make "doc" ]
+
+build: [ make "build" ]
+
+install: [ make "install" ]
+
+remove: [ "ocamlfind" "remove" "ISO8601" ]
+
+depends: [ "ocamlfind" ]
```

Comment:
`ocamlfind` should be marked as a build dependency as it is not used for the regular function of the package and an update to `ocamlfind` will not change the functionality or correctness of the package. This line should read something like `depends: [ "ocamlfind" {build} ]`.